### PR TITLE
Add `finished` property to `once_trigger`

### PR DESCRIPTION
**Pull request by hitsgub**

### CODE REVIEW

**Comment by hitsgub** [comment]
This PR will merge separated PRs #5565 and #5681.
The main target of `finished` property in #5681 is `once_trigger` in #5565, but the PR had been working.
Now their PRs were merged to master branch, so I add `finished` property to `once_trigger`.

File: tests/chainer_tests/training_tests/triggers_tests/test_once_trigger.py [hunk 1]
```diff
@@ -16,74 +16,94 @@ def get_expected(num, call_on_first=True):
...
+
 @testing.parameterize(
     # basic
     {
         'iter_per_epoch': 5, 'call_on_resume': False, 'resume': 4,
         'expected': get_expected(7),
-        'expected_resume': get_expected(7, False)},
+        'expected_resume': get_expected(7, False),
+        'finished': get_finished(7),
+        'finished_resume': get_finished(7, False)},
```
Comment (beam2d) [inline tests/chainer_tests/training_tests/triggers_tests/test_once_trigger.py]: I now think the parameterization is redundant. Is it right that the essential parameters are only `iter_per_epoch`, `call_on_resume`, and `resume`? How about removing others from parameter lists and making them properties instead?

Comment (hitsgub) [reply tests/chainer_tests/training_tests/triggers_tests/test_once_trigger.py]: Thank you.
Like this?

**Comment by beam2d** [comment]
Jenkins, test this please.

**Comment by beam2d** [comment]
Jenkins, test this please.

**Comment by beam2d** [comment]
Jenkins, test this please.

**Comment by beam2d** [comment]
Could you remove `import six`? LGTM except for that.

**Comment by beam2d** [comment]
Thanks.
Jenkins, test this please.

**Comment by beam2d** [comment]
LGTM

**Comment by beam2d** [comment]
Thanks!!

**Pull request merged**